docs/guides/email/02-basic-email-system.md

```diff
@@ -210,32 +232,30 @@ Mysql > insert into virtual_aliases(id,domain_id,source,destination) values(1,1,
 Mysql > insert into virtual_aliases(id,domain_id,source,destination) values(2,1,'all@mail.rockylinux.me','leeo@mail.rockylinux.me');
 ```
 
-Here I have not inserted the ciphertext password for the relevant email users, which requires the use of `doveadm pw -s SHA512-crypt -p twotestandtwo` command. See [here](#ap1)
+The password entries for the relevant email users are not shown here, as it requires the use of `doveadm pw -s SHA512-crypt -p twotestandtwo` command. See [here](#ap1)
```

Comment:
As a translator, I would prefer to put the command in a separate line, e.g., something similar to:

The password entries for the relevant email users are not shown here, as it requires the use of the following command:
```bash
doveadm pw -s SHA512-crypt -p twotestandtwo
```
 See [here](#ap1).

What is the meaning of the other translators?